tests/testthat/test-opart-gaussian.R

```diff
@@ -28,3 +28,20 @@ res <- opart_gaussian(c(1,2,3,4,5), 1000000)
 test_that("all the data points in one segment", {
   expect_equal(res$end.vec, 5)
 })
+
+#test for neuroblastoma data set with profile.id = 1 and chromosome = 1
```

Comment:
Hi @tdhock , I added neuroblastoma data for one profile id and one chromosome as a csv file for easy access and written tests for large and zero penalty.

Reply:
good but would be better to Suggests: neuroblastoma rather than copying the data set, see https://github.com/tdhock/penaltyLearning/blob/master/DESCRIPTION#L13 for an example